components/nodes/node-stanford-news.tsx

```diff
@@ -20,27 +25,63 @@ export const NodeStanfordNews = ({node, ...props}: NewsNodeProps) => {
         canonical={node.su_news_source?.url}
       />
 
-
-      <article {...props}>
+      <article {...props} className="su-mt-50">
         {node.su_news_topics && node.su_news_topics.map(topic =>
           <div key={topic.id}>
             {topic.name}
           </div>
         )}
 
-        <h1>{node.title}</h1>
-        {node.su_news_dek && <div>{node.su_news_dek}</div>}
-
-        {node.su_news_publishing_date && <div>{formatDate(node.su_news_publishing_date + ' 12:00:00')}</div>}
-        {node.su_news_byline && <div>{node.su_news_byline}</div>}
+        <h1 className="su-type-5">{node.title}</h1>
+        {node.su_news_dek && <div className="su-rs-mb-1">{node.su_news_dek}</div>}
+        <div className="md:su-flex su-rs-mb-7">
+          <div className="su-flex md:su-order-last su-rs-mb-2">
+            <ul className="su-flex su-list-unstyled md:su-pl-[10px] su-mt-[-3px]">
+              <li className="su-mr-1em">
+                <DrupalLink className="su-text-black hocus:su-text-facebook su-transition-colors" href="https://www.facebook.com">
+                  <span className="su-sr-only">Stanford Facebook</span>
+                  <FacebookIcon />
+                </DrupalLink>
+              </li>
+              <li className="su-mr-1em">
+                <DrupalLink className="su-text-black hocus:su-text-twitter su-transition-colors" href="https://twitter.com">
+                  <span className="su-sr-only">Stanford Twitter</span>
+                  <TwitterIcon />
+                </DrupalLink>
+              </li>
+              <li className="su-mr-1em">
+                <DrupalLink className="su-text-black hocus:su-text-linkedin su-transition-colors" href="https://www.linkedin.com/">
+                  <span className="su-sr-only">Stanford LinkedIn</span>
+                  <LinkedInIcon />
+                </DrupalLink>
+              </li>
+              <li className="su-mr-1em">
+                <DrupalLink className="su-text-black  hocus:su-text-digital-blue su-transition-colors" href="/">
+                  <span className="su-sr-only">Forward Email</span>
+                  <EnvelopeIcon className="su-w-[28px]"/>
+                </DrupalLink>
+              </li>
+              <li className="su-mr-1em">
+                <DrupalLink className="su-text-black  hocus:su-text-digital-blue su-transition-colors" href="/">
```

Comment:
This should have a print `onclick` like `onClick="window.print();return false;"`